docs/developers/authorization.md

```diff
@@ -0,0 +1,107 @@
+# Authorization Set up
+
+  Authorization is the process of giving someone permission to do or have something. In multi-user systems, a system administrator defines for the system which users are allowed access to the system and what privileges of use.
+
+We have a standard access control matrix with 3 axes:
+
+- Actions: CREATE, READ, WRITE, DELETE, PURGE
+- Entities (object): User, Publisher, Package, Package Resource, …
+- Users: a user or type of user
+
+## Introducing Roles
+
+It can be inefficient to list every user for every object. It is also cumbersome to manage in terms of business processes. For example:
```

Comment:
Suggest:

It can be tiresome and inefficient to list for every object all the users permitted to perform a given action.